tutorials/legacy/net/parallelMergeClient.C

```diff
@@ -25,15 +25,18 @@
 #include "TRandom.h"
 #include "TError.h"
 
-void parallelMergeClient()
+#include <string>
+
+void parallelMergeClient(const std::string &socketPath)
```

Comment:
We problably should update the documentation/running hints at lines 9,10 of this files as well as the corresponding doc at lines 17,18 of `parallelMergeServer.C`